tests/system/fim/common_tasks/verify_alerts_elasticsearch.py

```diff
@@ -76,8 +76,8 @@ def makeQuery(query, Elastic, index_name):
         help="Sleep time between retries", default="60"
     )
     parser.add_argument(
-        "-w", "--whodata", type=bool, required=False, dest='whodata_query',
-        help="Enable whodata queries", default="False"
```

Comment:
This is not correct, "False" is a string and is interpreted by Python3 as `True` when casting to a boolean.

Reply:
Why remove the `type` parameter?

Reply:
I checked `default=False` without quotes and returned False, also when declaring `-w True` returned True.

Please consider leaving `type` parameter and setting the default to `False`

Reply:
Yes, default=False should do the trick, but the correct what to store booleans using arparse and flags is using `store_true` or `store_false`. These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.

Reply:
` but the correct what to store booleans using arparse and flags is using store_true or store_false` Please link some references so we can have as an argument to propagate this change to others scenarios. Thanks! :+1:

Reply:
Sure, check https://docs.python.org/3/library/argparse.html
As I said, `default=False` with `type` will work but that library has a special way to add booleans (search for `store_true` for more info) and using that is not required to specify the boolean type

Reply:
I read related documentation, you are completely right, thanks for the references!